packages/strapi-generate-api/templates/mongoose/service.template

```diff
@@ -37,7 +37,7 @@ module.exports = {
 
   fetch: (params) => {
     return <%= globalID %>
-      .findOne(_.pick(params, _.keys(<%= globalID %>.schema.path)))
+      .findOne(_.pick(params, _.keys(<%= globalID %>.schema.paths)))
```

Comment:
```js
.findOne(_.pick(params, _.keys(<%= globalID %>.attributes)))
```
Why are we using the native Mongoose schema instead of our layer in this case? If the Mongoose library decides to change the key, it will break again. I consider it should be better to use our models.

However, it will limit the search only on the declared attributes in our models.

Reply:
Write, but actually `id` is not in attributes.